src/util/upnp_clients.cc

```diff
@@ -156,7 +156,7 @@ void Clients::getInfo(const struct sockaddr_storage* addr, const std::string& us
     // by addClientByDiscovery
     if (!found) {
         // always return something, 'Unknown' if we do not know better
-        assert(clientInfo[0].type == ClientType::Unknown);
+        static_assert(clientInfo[0].type == ClientType::Unknown);
```

Comment:
I think we want to assert at runtime still

Reply:
Why?

Reply:
Because the check is a runtime one and makes no sense at compile time?

Reply:
The way I read this is that it asserts that the first struct member at 0 is Unknown, which it is. Either that or the check is wrong.

Reply:
Yes, but why would we want to check that at compile time?

Reply:
It’s a const struct...